Detectors/GlobalTrackingWorkflow/qc/src/ITSTPCMatchingQCSpec.cxx

```diff
@@ -31,20 +31,25 @@ namespace globaltracking
 
 void ITSTPCMatchingQCDevice::init(InitContext& /*ic*/)
 {
-  const o2::globaltracking::ITSTPCMatchingQCParams& params = o2::globaltracking::ITSTPCMatchingQCParams::Instance();
+  const o2::gloqc::ITSTPCMatchingQCParams& params = o2::gloqc::ITSTPCMatchingQCParams::Instance();
 
-  mMatchITSTPCQC = std::make_unique<o2::globaltracking::MatchITSTPCQC>();
+  mMatchITSTPCQC = std::make_unique<o2::gloqc::MatchITSTPCQC>();
   mMatchITSTPCQC->init();
   mMatchITSTPCQC->setDataRequest(mDataRequest);
   mMatchITSTPCQC->setPtCut(params.minPtCut);
   mMatchITSTPCQC->setEtaCut(params.etaCut);
   mMatchITSTPCQC->setMinNTPCClustersCut(params.minNTPCClustersCut);
   mMatchITSTPCQC->setMinDCAtoBeamPipeDistanceCut(params.minDCACut);
   mMatchITSTPCQC->setMinDCAtoBeamPipeYCut(params.minDCACutY);
+  mMatchITSTPCQC->setCutK0Mass(params.cutK0Mass);
+  mMatchITSTPCQC->setMaxK0Eta(params.maxEtaK0);
   o2::base::GRPGeomHelper::instance().setRequest(mCCDBRequest);
   if (mUseMC) {
     mMatchITSTPCQC->setUseMC(mUseMC);
   }
+  if (mDoK0QC) {
```

Comment:
technically also for the mUseMC

Reply:
Isn't it clearer?

Reply:
I don't think it is, since these settings come from the command line and are just passed on. This only works since the default settings in 'mMatchITSPTCQC' are reasonable. This kind of pattern disconnects settings coming from one place (cli) and the place where they are used. Anyways, this of course works so its fine was just an observation :)